app/response/utils.py

```diff
@@ -424,6 +456,29 @@ def process_email_template_request(request_id, data):
                                agency_name=agency_name,
                                note_content=note_content,
                                page=page)
+    # process email template for offline instructions
+    if data['type'] == 'instruction_email':
+        # if data['instructions'] exists, use email_content as template with specific instructions template
+        try:
+            instruction = data['instruction']
+            default_content = False
+            content = data['email_content']
+            if instruction['privacy'] != PRIVATE:
+                instruction_content = instruction['content']
+            else:
+                instruction_content = ''
```

Comment:
What do you think about `instruction_content = instruction['content'] if instruction['privacy'] != PRIVATE else ''`?